+Prefs/+Inputs/LabelControlBasic.m

```diff
@@ -51,7 +51,8 @@
                         'horizontalalignment','left',...
                         'units', 'pixels',...
                         'tag', tag,...
-                        'enable', enabled);
+                        'enable', enabled,...
+                        'UserData', enabled);
```

Comment:
Can you comment as to why?

Reply:
Putting comment in code. When displaying NaN, I overwrite `Enable` to `false`, i.e. overwrite `read_only`. I wanted to have another copy of `read_only` in the UI.